frontend/src/scenes/instance/SpecialMigrations/index.tsx

```diff
@@ -0,0 +1,224 @@
+import './index.scss'
+
+import React from 'react'
+import { PageHeader } from 'lib/components/PageHeader'
+import { SceneExport } from 'scenes/sceneTypes'
+import { Button, Modal, Progress, Space, Table } from 'antd'
+import { useActions, useValues } from 'kea'
+import { SpecialMigration, migrationStatusNumberToMessage, specialMigrationsLogic } from './specialMigrationsLogic'
+import {
+    PlayCircleOutlined,
+    StopOutlined,
+    RedoOutlined,
+    CheckCircleOutlined,
+    InfoCircleOutlined,
+} from '@ant-design/icons'
+import { humanFriendlyDetailedTime } from 'lib/utils'
+import { Tooltip } from 'lib/components/Tooltip'
+import { Spinner } from 'lib/components/Spinner/Spinner'
+import { userLogic } from 'scenes/userLogic'
+
+export const scene: SceneExport = {
+    component: SpecialMigrations,
+}
+
+export const tooltipMessageForStatus = {
```

Comment:
maybe add a comment linking to MigrationStatus so we know what these magic numbers are more easily